experiment/assignment.md

```diff
@@ -0,0 +1,56 @@
+### Practice Assignment
+
+1. **Reversing a String** (Beginner)
```

Comment:
Sir, yeah these questions can be solved without recursion too, but then it would be difficult to find questions which can be done using recursion. Moreover, the questions which can be solved only using recursion can become difficult. Also, there was a special motive behind giving some of these questions, as they would know the basic logic behind what is to be done, but then if they use recursion for it, they can easily grasp the behind-the-scene understanding of the topic (recursion). Usually, in beginners courses, it is done.

Reply:
Maybe you can ask them to do the question whichever way they find intuitive and then try to do the same question using recursion. The idea is to get them thinking in the recursion direction.

Reply:
I have added a note for this same thing.